crates/spirv-std/src/arch.rs

```diff
@@ -150,3 +150,26 @@ pub unsafe fn vector_insert_dynamic<T: Scalar, V: Vector<T, N>, const N: usize>(
 pub fn kill() -> ! {
     unsafe { asm!("OpKill", options(noreturn)) }
 }
+
+/// Read from the shader clock.
+///
+/// Requires the `SPV_KHR_shader_clock` extension and the `ShaderClockKHR` capability.
+///
+/// See:
+/// <https://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/KHR/SPV_KHR_shader_clock.html>
+#[spirv_std_macros::gpu_only]
+#[doc(alias = "OpReadClockKHR")]
+pub unsafe fn read_clock_khr() -> u64 {
```

Comment:
I think this should have a `#[cfg(target_feature = "Int64")]`. It would be nice to also include the u32-tuple version, to allow this to be used where the Int64 capability is not available.